### DISCUSSION TITLE
How to view active clock ins

### DISCUSSION THREAD

**Discussion by Advaera** [kimai/kimai#3139] [Questions & Answers] [answered]
Good afternoon!
Is there a way to view a list of users who are actively clocked in? 
Maybe something like a report that shows the number of users who are clocked in over the total number of users and then lists all user names who are actively clocked in?

If Kimai doesn't do this by default, is there a known plugin that might add this feature?

Thank you very much!

**Comment by kevinpapst** [comment, marked as answer]
There is no such report yet.

But you can filter all running timesheets in `Admin > Timesheet`.

**Reply by Advaera** [reply]
Thank you for the quick response!

**Reply by kevinpapst** [reply]
Sorry there is no better feedback, but I will add such a report in the future.
Definitely interesting for companies with multiple employees!

**Reply by Advaera** [reply]
Looking forward to that! There are a few report customizations that would be great to have such as a print view and having options to list times clocked in and out as well as total hours. All in all, we're very impressed with Kimai. Please keep being amazing!
